src/display/annotation_layer.js

```diff
@@ -577,8 +579,7 @@ class WidgetAnnotationElement extends AnnotationElement {
     for (const [baseName, eventName] of names) {
       if (
         eventName === "Action" ||
-        (this.data.actions &&
-          this.data.actions[eventName.replace(" ", "")] !== undefined)
+        (this.data.actions && this.data.actions[eventName] !== undefined)
```

Comment:
Would is suffice to just check the following instead?
```suggestion
        this.data.actions?.[eventName]
```